packages/node/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "sourceRoot": ".",
+    // TODO: Document why we need declarationDir here
+    "declarationDir": "dist/",
```

Comment:
Do we need `declarationDir`?

Reply:
I do not think we do. No package consumes its typings.